### PR TITLE
Overall adjustments

### PR DESCRIPTION
**Pull request by cvriel**
- constant DATASTORE_TYPE moved to common
- replaced all quote methods in DAG's with common quote_string
- used psycopg2 in SQL statement financien_grootboek
- make return connection in common

### CODE REVIEW

File: src/dags/common/db.py [hunk 1]
```diff
@@ -19,6 +19,12 @@ def __init__(self, postgres_conn_id="postgres_default"):
         self.db = self.connection.schema
 
 
+def get_connection(postgres_conn_id: str = "postgres_default") -> Engine:
```
Comment (gkoller) [inline src/dags/common/db.py:22]: This type annotation does not seem correct. `Engine` is not in the type hierarchy of `PostgresHook`

20210607_083135_screenshot

File: src/dags/common/db.py [hunk 2]
```diff
@@ -19,6 +19,12 @@ def __init__(self, postgres_conn_id="postgres_default"):
...
+    """Return database connection."""
+    connection = PostgresHook(postgres_conn_id=postgres_conn_id)
```
Comment (gkoller) [inline src/dags/common/db.py:24]: Naming the `PostgresHook` `connection` is kind of confusing. `PostgresHook` is not a connection, it encapsulates one, namely `psycopg2._psycopg.connection.connection` to be retrieved using `PostgresHook`'s `get_conn` method.

What I'm saying that the message that this function conveys is rather ambiguous:
- the function is called `get_connection`
- that does not *seem* to return a connection, but an `Engine`
- though the implementation, although the instance is named `connection`, in reality returns a `PostgresHook` instance.

**Review by gkoller — APPROVED** [review]
Nice clean up. One remaining issue re: `common.db.get_connection`

File: src/dags/common/db.py (hunk 1 above)
Comment (cvriel) [reply src/dags/common/db.py:22]: Oeps, pas ik aan

File: src/dags/common/db.py (hunk 2 above)
Comment (cvriel) [reply src/dags/common/db.py:24]: You are right, the get_connection is confusing and does not fit. I will change it's name to be more appropriate

**Pull request closed without merging**